cellpose/gui/guiparts.py

```diff
@@ -490,6 +490,25 @@ def update_minimap(self, parent):
         else:
             self.setFixedSize(400, 400)
 
+    def mousePressEvent(self, event):
+        """
+        Handles mouse press events on the minimap.
+        This method is triggered when the user clicks on the minimap,
+        allowing for interaction such as navigating the main image view.
+        """
+        # Obtain the position where the mouse was clicked within the minimap.
+        viewboxPos = event.pos()
+
+        # Save the translated position for later use
+        self.lastClickPos = (viewboxPos.x(), viewboxPos.y())
+
+        # Normalize the clicked position's coordinates to values between 0 and 1.
+        normalized_x = viewboxPos.x() - 9/ self.viewbox.width()
+        normalized_y = viewboxPos.y() - 9/ self.viewbox.height()
+        self.normalizedClickPos = (normalized_x, normalized_y)
+
+        # Update the position of the minimap
+        self.viewbox.autoRange()

```

Comment:
Is this a spaceholder? There should be no updating of the minimap viewbox in this method imo, only of the mainW viewbox. This is part of ticket 46.